src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -33,7 +35,9 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
 
         this.addressBook = new AddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
-        filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        this.filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        this.filteredPolicies = new FilteredList<>(this.addressBook.getPolicyList());
```

Comment:
Actually whats the purpose of this line ah? the FilteredPerson has this line cus when the app fires up, it needs to display every person in the UI. Then subsequently, when user calls the findCommand. It uses the updateFiltedPersonList to update the list shown to the user in the UI.

Reply:
I need it so i can pass in a filtering predicate to get policies within 30 days of expiry